src/modules/twap/utils/getPartOrderStatus.ts

```diff
@@ -10,14 +10,23 @@ export function getPartOrderStatus(
   parent: TwapOrderItem,
   isVirtualPart: boolean
 ): OrderStatus {
+  const isExpired = isOrderExpired(enrichedOrder)
+  const isCancelled = isOrderCancelled(enrichedOrder) || enrichedOrder.status === SdkOrderStatus.CANCELLED
```

Comment:
Isn't the second part of the conditional covered in `isOrderCancelled`? If not, shouldn't it be?

Reply:
Actually yes, we can put the second condition inside of the function. Thanks!